hazelcast/src/main/java/com/hazelcast/spi/discovery/impl/DefaultDiscoveryService.java

```diff
@@ -133,14 +134,25 @@ private boolean validateCandidate(DiscoveryNode candidate) {
         try {
             Collection<DiscoveryStrategyConfig> discoveryStrategyConfigs = new ArrayList<DiscoveryStrategyConfig>(
                     settings.getAllDiscoveryConfigs());
-
             List<DiscoveryStrategyFactory> factories = collectFactories(discoveryStrategyConfigs, configClassLoader);
 
             List<DiscoveryStrategy> discoveryStrategies = new ArrayList<DiscoveryStrategy>();
             for (DiscoveryStrategyConfig config : discoveryStrategyConfigs) {
                 DiscoveryStrategy discoveryStrategy = buildDiscoveryStrategy(config, factories);
                 discoveryStrategies.add(discoveryStrategy);
             }
+
+            if (discoveryStrategies.isEmpty() && settings.isAutoDetectionEnabled()) {
+                logger.info("Discovery auto-detection enabled, looking for available discovery strategies");
```

Comment:
probably should not to info, we are trying to reduce logging noise. see: https://github.com/hazelcast/hazelcast/pull/17243

Reply:
Changed this one. But I'd keep info for the following lines because it's very important to users why other members are discovered. PTAL.